src/app/home/home.component.html

```diff
@@ -71,8 +71,10 @@ <h5>App Development</h5>
     <!-- End of what we do -->
 
     <!-- Start of Solutions -->
-    <h1 class="animate__animated animate__fadeInUp animate__backOutLeft solution text-center" style="font-family: Playfair Display;">Solutions</h1>
-    <div style="background-image: url('../../assets/solutions_block_bg.png'); background-size: contain; background-position: center center; background-repeat: no-repeat;">
+    <h1 class="animate__animated animate__fadeInUp animate__backOutLeft solution text-center"
```

Comment:
Remove the dots and place like this. /assets

Reply:
updated assets paths